lms/static/js/search/app.js

```diff
@@ -0,0 +1,35 @@
+
+var edx = edx || {};
```

Comment:
In a browser, the global scope (this) is  the window object. So, except for older IEs, those two statements should be equivalent.